Marek Mahut <mmahut@fedoraproject.org>: This patch fixes the build when libtool fails to detect that lib

diff -Naur gnuradio-3.2-upstream/gnuradio-core/src/tests/Makefile.am gnuradio-3.2/gnuradio-core/src/tests/Makefile.am
--- gnuradio-3.2-upstream/gnuradio-core/src/tests/Makefile.am	2009-07-25 09:14:01.090447704 +0200
+++ gnuradio-3.2/gnuradio-core/src/tests/Makefile.am	2009-07-25 09:25:12.681195057 +0200
@@ -59,7 +59,7 @@
 	benchmark_dotprod
 
 
-LIBGNURADIO = 	$(GNURADIO_CORE_LA)
+LIBGNURADIO = 	$(GNURADIO_CORE_LA) $(top_builddir)/omnithread/libgromnithread.la
 LIBGNURADIOQA = $(top_builddir)/gnuradio-core/src/lib/libgnuradio-core-qa.la $(LIBGNURADIO)
 
 benchmark_dotprod_fff_SOURCES = benchmark_dotprod_fff.cc
